apps/gp2-server/src/controllers/user.controller.ts

```diff
@@ -39,24 +34,22 @@ export default class Users implements UserController {
   async update(
     id: string,
     update: gp2.UserUpdateRequest,
-    loggedInUserId: string,
   ): Promise<gp2.UserResponse> {
     await this.userDataProvider.update(id, update);
-    return this.fetchById(id, loggedInUserId);
+    return this.fetchById(id, id);
```

Comment:
Does fetchById function need also to receive both id's? It seems that it's not used anywhere else except in user.route.ts, but could be mistaken

Reply:
when we parse the squidex data to our UserResponse, we check if the user making the request is the one being fetched and if it is true we will add telephone data, else we don't (for security/privacy concerns)
we we make a update we always returned the user, but for an update we already validate you are the user we are changing